drake/examples/Atlas/+atlasFrames/AtlasCoordinates.m

```diff
@@ -4,7 +4,8 @@
     function obj=AtlasCoordinates(r)
       typecheck(r,'TimeSteppingRigidBodyManipulator');
       nq = r.getNumPositions();
-      obj = obj@SingletonCoordinateFrame('atlasFrames.AtlasCoordinates',nq,'x',r.getStateFrame.coordinates(1:nq));
+      coords = r.getStateFrame();
+      obj = obj@SingletonCoordinateFrame('atlasFrames.AtlasCoordinates',nq,'x',coords(1:nq));
```

Comment:
It's annoying that slicing the coordinate names can no longer be done as a one-liner. But that's unavoidable (unless Mathworks fixes https://github.com/RobotLocomotion/mathworks-requests/issues/8 :wink: )